### PR TITLE
[TR] PIM-5273: Add doc for Presenters

### PR DESCRIPTION
**Pull request by pierallard**
`how_to_use_localizers` is not a part of this PR, it's just a failed git mv.

### CODE REVIEW

File: cookbook/localization/how_to_use_the_localizers.rst [hunk 1]
```diff
@@ -0,0 +1,64 @@
+How to Use the Localizers
```
Comment (MarieMinasyan) [inline cookbook/localization/how_to_use_the_localizers.rst]: sans "the"

File: cookbook/localization/how_to_use_the_localizers.rst [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+=========================
+
+Localizers are used to convert a localized attribute to the standard format.
+E.g. a decimal with a comma separator have to be convert with a dot separator before being inserted in database.
```
Comment (MarieMinasyan) [inline cookbook/localization/how_to_use_the_localizers.rst]: has to be converted to... inserted into the database.

File: cookbook/localization/index.rst [hunk 3]
```diff
@@ -1,63 +1,15 @@
-How to Use the Localizers
+How to Localize your data
 =========================
 
-Localizers are used to convert a localized attribute to the standard format.
-E.g. a decimal with a comma separator have to be convert with a dot separator before being inserted in database.
+Since the 1.5 version, Akeneo PIM can be used simultaneously by people of different language in the same installation.
```
Comment (MarieMinasyan) [inline cookbook/localization/index.rst]: languages

File: cookbook/localization/index.rst [hunk 4]
```diff
@@ -1,63 +1,15 @@
...
+This implies that it is necessary to manage language-related specificities.
+For example, the English decimal separator is a dot (.), the French one it is a comma (,).
```
Comment (MarieMinasyan) [inline cookbook/localization/index.rst]: the French one is

File: cookbook/localization/how_to_use_the_presenters.rst [hunk 5]
```diff
@@ -0,0 +1,42 @@
+How to use Presenters
+=====================
+
+The Presenters are used to display the data for the end user according to their local or other parameters.
```
Comment (skeleton) [inline cookbook/localization/how_to_use_the_presenters.rst]: or other parameters ?

Comment (pierallard) [inline cookbook/localization/how_to_use_the_presenters.rst]: @skeleton Here we only use presenters to present data from a specific locale ; the structure of the presenter can be used to have any option (anything)

File: cookbook/localization/how_to_use_the_localizers.rst [hunk 6]
```diff
@@ -0,0 +1,64 @@
...
+.. code-block:: yaml
+
+    pim_localization.localizer.my_own_localizer:
+        class: %my_localizer_class%
+        arguments:
+            - ['pim_catalog_number']
+        tags:
+            - { name: pim_localization.localizer }
+
+The only argument of the service is the list of attribute types you want to convert.
```
Comment (phaseinducer) [inline cookbook/localization/how_to_use_the_localizers.rst]: "supported by your localizer" ?

**Comment by ClementGautier** [comment]
GTM

File: cookbook/localization/how_to_use_the_presenters.rst [hunk 7]
```diff
@@ -0,0 +1,45 @@
...
+
+Actually, the Presenter interface displays data according to the user locale, but you can create a Presenter using custom options.
+
+
+How to modify the predefined formats of default Presenters
+----------------------------------------------------------
+
+You can define your own formats to present dates, datetimes and prices depending on the user's locale.
+The formats of date, datetimes and currencies are predefined for a set of languages.
+When the requested format is not predefined, the fallback is defined by http://php.net/manual/en/class.intldateformatter.php and http://php.net/manual/en/class.numberformatter.php .
```
Comment (phaseinducer) [inline cookbook/localization/how_to_use_the_presenters.rst]: Would be cleaner to put labels for links here

File: cookbook/localization/how_to_use_the_presenters.rst [hunk 8]
```diff
@@ -0,0 +1,45 @@
...
+
+To add or modify the default formats, you have to add or modify the `pim_localization.factory` tags (`date.formats`, `datetime.formats` and `currency.formats`). For example:
+
+.. code-block:: yaml
+
+    pim_localization.factory.currency.formats:
+        en_US: '¤#,##0.00'
```
Comment (phaseinducer) [inline cookbook/localization/how_to_use_the_presenters.rst]: Encoding issue ?

**Comment by phaseinducer** [comment]
Some comments & GTM

File: cookbook/localization/how_to_use_the_presenters.rst (hunk 8 above)
Comment (pierallard) [inline cookbook/localization/how_to_use_the_presenters.rst]: Nope !

**Pull request merged**